src/language/dafnyInstallation.ts

```diff
@@ -57,8 +57,10 @@ export function getLanguageServerRuntimePath(context: ExtensionContext): string
   return path.join(context.extensionPath, configuredPath);
 }
 
-function getConfiguredLanguageServerRuntimePath(): string | null {
-  return Configuration.get<string | null>(ConfigurationConstants.LanguageServer.RuntimePath);
+function getConfiguredLanguageServerRuntimePath(): string {
+  const languageServerOverride = process.env['DAFNY_LANGUAGE_SERVER'] ?? '';
+  const languageServerSetting = Configuration.get<string | null>(ConfigurationConstants.LanguageServer.RuntimePath) ?? '';
+  return languageServerOverride || languageServerSetting;
```

Comment:
I'm surprised that you don't have a lint warning here, but, hey, that's good that it works, I like it!

Reply:
That's because the type is `string` instead of `string | null`